arkouda/strings.py

```diff
@@ -354,6 +357,72 @@ def get_lengths(self) -> pdarray:
             generic_msg(cmd="segmentLengths", args={"objType": self.objType, "obj": self.entry})
         )
 
+    def get_bytes(self):
+        """
+        Property accessor that returns a pdarray of bytes (uint8) of the
+        corresponding accessed Strings or SegString.
```

Comment:
Not super important but this doc string isn't accurate anymore. Since this a getter not a property accessor now. Also we don't have to use such vague language of `corresponding accessed Strings` because this called on a specfic Strings (i.e. `my_str.get_bytes`) so we know it's this Strings that we want the bytes component of. Does that make sense?
```suggestion
        Getter for the bytes component (uint8 pdarray) of this Strings.
```